### ISSUE TITLE
Style UI: Add an autocomplete search box on the documentation column

### ISSUE THREAD

**Issue by franckl** [mapbox/mapbox-studio-classic#1044]
This search box would filter the cartocss properties

**Comment by samanpwbb** [comment]
Yes, feasible. I am a little wary about implementing this, since built-in browser cmd + f works pretty decently for searching cartocss.

**Comment by franckl** [comment]
I am using the os x app. I didn't know you could get it in the browser

**Comment by samanpwbb** [comment]
@franckl ahh, you're right. Node-atom doesn't provide a search bar. This certainly helps your case.
